src/components/ViewerTitle/index.tsx

```diff
@@ -36,7 +36,11 @@ const ViewerTitle: React.FunctionComponent<ViewerTitleProps> = (
     const version = currentTrajectory ? currentTrajectory.version : "";
 
     let tagText = "";
-    if (lastModified) {
+
+    // lastModified is a date in the future if file was loaded via external link in URL param
```

Comment:
the comment made me think that it happened by external means as opposed to your code doing it intentionally

Reply:
Thanks for pointing it out, I made it more verbose and hopefully less confusing